docs/internal/contributing/README.md

```diff
@@ -56,12 +56,57 @@ make docker-image/pyroscope/build
 This target uses the `go/bin` target to first build binaries to include in the image.
 Make sure to pass the correct `GOOS` and `GOARCH` env variables.
 
-For example for `linux/amd64`:
-
+#### amd64 builds
 ```
 GOOS=linux GOARCH=amd64 make docker-image/pyroscope/build
 ```
 
+#### arm64 builds
+```
+GOOS=linux GOARCH=arm64 make docker-image/pyroscope/build
```

Comment:
This would build a linux/amd64 image with a linux/arm64 binary, so you need this instead:

Note: also the make command doesn't work for me on environment variables so make needs to come first.

```suggestion
make IMAGE_PLATFORM=linux/arm64 GOOS=linux GOARCH=arm64 docker-image/pyroscope/build
```